web/src/index.tsx

```diff
@@ -1,19 +1,36 @@
-import React from 'react';
-import ReactDOM from 'react-dom';
-import './index.css';
-import App from './App';
-import reportWebVitals from './reportWebVitals';
+import React from "react";
+import ReactDOM from "react-dom";
+import "./index.css";
+import App from "./App";
+import reportWebVitals from "./reportWebVitals";
 
-import { AppContextProvider } from './GlobalState'
+// 1
+import {
+    ApolloProvider,
+    ApolloClient,
+    createHttpLink,
+    InMemoryCache,
+} from "@apollo/client";
 
+// 2
+const httpLink = createHttpLink({
+    uri: "http://localhost:8000/graphql",
```

Comment:
Can this just be `/graphql`:

```suggestion
    uri: "/graphql",
```

Reply:
Fixed